Core/Jira/CredentialManagement/JiraCredentialManager.cs

```diff
@@ -64,20 +66,17 @@ public Credentials GetCredential (string target)
     catch
     {
       _console.WriteLine("Invalid Jira Credentials saved.");
-      return AskForCredentials(target);
+      return AskForBasicCredentials(target, userInput);
     }
 
     return credentials;
   }
 
-  private Credentials AskForCredentials (string target)
+  private Credentials AskForBasicCredentials (string target, Func<Credentials> askForUserInput)
```

Comment:
Shouldn't this be "AskForCredentions since the delegate decides?
also, since we already use a field, couldn't we move the func into the AskForCredentials method instead of using a parameter and inside, just call the methodsw instead of using a func?

Reply:
A yes, remnants of an old implementation idea which was quickly overhauled ;D thanks for the find